<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>64</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/EngineButtonStart.png</normaloff>:/img/EngineButtonStart.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QwtCompass" name="YawCompass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>187</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="frameShadow">
    <enum>QwtDial::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/Drone-Top.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="yawDial">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>303</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="value">
    <number>180</number>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="headingLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HEADING</string>
   </property>
  </widget>
  <widget class="QwtCompass" name="RollCompass">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>184</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="RollLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>N - S</string>
   </property>
  </widget>
  <widget class="QLabel" name="NoPitchLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>m/s</string>
   </property>
  </widget>
  <widget class="QSlider" name="rollSlider">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>130</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QwtCompass" name="PitchCompass">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>186</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>253</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/DronePerfil.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="PitchLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>161</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>W - E</string>
   </property>
  </widget>
  <widget class="QLabel" name="drone">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>236</y>
     <width>72</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/PITCH.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="pitchSlider">
   <property name="geometry">
    <rect>
     <x>699</x>
     <y>232</y>
     <width>20</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-45</number>
   </property>
   <property name="maximum">
    <number>45</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>96</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="RollLabel_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QLabel" name="RollLabel_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QLabel" name="PitchLabel2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QLabel" name="PitchLabel3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>260</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QLabel" name="Angulo_Label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>280</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Angulo </string>
   </property>
  </widget>
  <widget class="QLabel" name="swheel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/Volante.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Volante</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Avion</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>YawCompass</zorder>
  <zorder>runButton</zorder>
  <zorder>splitter_2</zorder>
  <zorder>label_2</zorder>
  <zorder>yawDial</zorder>
  <zorder>headingLabel</zorder>
  <zorder>RollCompass</zorder>
  <zorder>RollLabel</zorder>
  <zorder>NoPitchLabel</zorder>
  <zorder>rollSlider</zorder>
  <zorder>PitchCompass</zorder>
  <zorder>label_3</zorder>
  <zorder>PitchLabel</zorder>
  <zorder>drone</zorder>
  <zorder>pitchSlider</zorder>
  <zorder>lcdNumber</zorder>
  <zorder>RollLabel_3</zorder>
  <zorder>RollLabel_4</zorder>
  <zorder>PitchLabel2</zorder>
  <zorder>PitchLabel3</zorder>
  <zorder>Angulo_Label</zorder>
  <zorder>swheel</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtCompass</class>
   <extends>QwtDial</extends>
   <header>qwt_compass.h</header>
  </customwidget>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>runButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
